src/app.html

```diff
@@ -34,7 +34,8 @@
 
 		%sveltekit.head%
 	</head>
+
 	<body>
-		%sveltekit.body%
+		<div style="display: contents">%sveltekit.body%</div>
```

Comment:
Why is this needed?

Reply:
> `%sveltekit.body%` — the markup for a rendered page. This should live inside a `<div>` or other element, rather than directly inside `<body>`, to prevent bugs caused by browser extensions injecting elements that are then destroyed by the hydration process. SvelteKit will warn you in development if this is not the case
https://kit.svelte.dev/docs/project-structure#project-files-src

Reply:
Oh, TIL! Nice catch 😄